digitaltwin_client/inc/digitaltwin_device_client.h

```diff
@@ -51,10 +51,11 @@ typedef void* DIGITALTWIN_DEVICE_CLIENT_HANDLE;
   @param    deviceHandle[in]            An IOTHUB_DEVICE_CLIENT_HANDLE that has been already created and bound to a specific connection string (or transport, or DPS handle, or whatever
                                         mechanism is preferred).  See remarks about its lifetime management.
   @param    dtDeviceClientHandle[out]   A <c>DIGITALTWIN_DEVICE_CLIENT_HANDLE</c> to be used by the application.
- 
+  @param    deviceCapabilityModel[in]   The device's Device Capability Model (DCM) identifier. 
```

Comment:
>device's [](start = 44, length = 8)

Nit: I think "device's Device Capability Model" could just be "Device Capability Model".